lib/src/pub_summary.dart

```diff
@@ -13,15 +13,13 @@ import 'utils.dart';
 
 class PubSummary {
   static final _solvePkgLine = new RegExp(
-      r"(?:[><\+ ]) (\w+) (\S+)(?: \((\S+) available\))?(?: from .+)?");
+      r"(?:[><\+\! ]) (\w+) (\S+)(?: \((\S+) available\))?(?: from .+)?");
```

Comment:
Oops! Didn't handle the override case. Doing the validation actually helped me fix a bug